### DISCUSSION TITLE
Vue 3 support not released

### DISCUSSION THREAD

**Discussion by rjd22** [uNmAnNeR/imaskjs#530] [Q&A]
**What problem you are trying to solve?**
I'm trying to use the `vue-imask` component with vue3 but is isn't working. Most likely it has to do with that the fixed made in the following commit were not released yet:

https://github.com/uNmAnNeR/imaskjs/commit/852495501e611bc834af91af920b62c96d0b340f

**Describe the solution you'd like**
Would it be possible to create a new release or alpha release with the new changes so we can use this component together with vue3?

**Comment by uNmAnNeR** [comment]
Hey! i am trying to add typings to plugins (especially for React and vue).
I started from react and did configure typescript. But for now do not have much time to fix types warnings.
Supposed it will be an easy but...

**Reply by rjd22** [reply]
I think typing is cool also but it's a optional feature for JavaScript.

I would recommend tagging a pre-release so people know it's unfinished but can start using it already if they don't care about typing.

**Comment by awacode21** [comment]
News on this?

**Reply by uNmAnNeR** [reply]
it were released actually
